test/840_weave_kube_3_test.sh

```diff
@@ -52,6 +52,12 @@ assert_raises "run_on $HOST1 $PING $HOST2EXPIP"
 assert_raises "run_on $HOST2 $PING $HOST1EXPIP"
 assert_raises "run_on $HOST3 $PING $HOST2EXPIP"
 
+# Check launch.sh does not stick around as a defunct process after having started weaver:
+count_defunct_launch_processes="ps aux | grep 'launch.sh' | grep defunct | wc -l"
```

Comment:
can we just check for any defunct processes, i.e. not require them to be named `launch.sh`?

also `grep -c` would elide `wc -l`